assets/javascripts/helpers/ifCond.js.es6

```diff
@@ -0,0 +1,17 @@
+//This helper class is performing a greater than operation on given two values and returning True or False
+// based on the evaluation
+import { registerHelper } from 'discourse-common/lib/helpers';
+
+
+registerHelper('ifCond', function(params) {
+    let topicValueIsGreaterThanTheBoundaryRiskLevel = false;
+    let topicValue = params[0];
+    let operator = params[1];
+    let boundaryRiskLevel = params[2];
+    if (operator === '>'){
+        if (topicValue > boundaryRiskLevel) {
```

Comment:
I wonder if we want to guard the numeric values just in case something changes in the future.  i.e. force or coerce them to be numbers at assignment in the `let` statements, so that in the future if they start coming in as strings it will still work